sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -198,10 +199,10 @@ static final DataType getSparkType(DType type) {
     }
   }
 
-  protected static final DataType getSparkTypeFrom(ColumnViewAccess access) {
+  protected static final DataType getSparkTypeFrom(ColumnViewAccess<BaseDeviceMemoryBuffer> access) {
```

Comment:
Will we never need to call this method from host column vectors?

Reply:
Adding this call to host side is not on the cards right now. Also, since this call is coming from column vector, which will always have device memory buffers at least for now, I made this change under this assumption.

Reply:
Maybe I can just look at ignoring this error. To be safe.

Reply:
Yeah we could add a warning suppression to allow this to be called from the host side in the future.

Reply:
Sounds good. You got it! Thanks Jason. Updating soon.

Reply:
You could also make the method generic and not worry about it.

```
protected static final <T> DataType getSparkTypeFrom(ColumnViewAccess<T> access) { ...
```

Reply:
Sorry I was out sick yesterday. Sounds good Bobby. Will update in a couple minutes here.